Gemfile

```diff
@@ -16,6 +16,7 @@ gem "mongoid"
 gem "plek"
 gem "responders"
 gem "sass-rails"
+gem "sprockets-rails"
```

Comment:
Do we need to bundle `sprockets-rails` explicitly? I can see it is already a dependancy of the app due to [rails](https://github.com/alphagov/imminence/blob/main/Gemfile.lock#L384) and [sassc_rails]( https://github.com/alphagov/imminence/blob/main/Gemfile.lock#L469), with the version installed being the latest release (3.4.2).

Reply:
Hmm, good question - according to the Rails 7 upgrade instructions we're following, sprockets _isn't_ a dependency of rails any more. But perhaps that's changed. I'll check.

Reply:
As discussed, going to add a commit note here to show our thinking: the sprockets-rails gem is a dependency of sassc-rails, but if in future we take sassc-rails out, we'd have to know to put the sprockets-rails gem dependency back in. Since we're not 100% sure how that missing dependency would manifest itself (safely at dev/test time or unsafely later on), safer to leave it in as an explicit dependency.